smokey/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -469,6 +470,16 @@ public void zeroGyroscope() {
       System.out.println("WARNING: Gyro is calibrating. Zeroing gyro has no effect while it is calibrating.");
     }
     swerveNavx.zeroYaw();
+    this.yawOffsetDegrees = 0.0;
+  }
+
+  /**
+   * sets the Yaw to a specific angle
+   * @param offsetDegrees
+   */
+  public void setYaw(double offsetDegrees) {
+    this.zeroGyroscope();
```

Comment:
Curious to know why we zero and then update the yaw offset?

Reply:
For zero gyroscope, we just need to both zeroYaw and set our offset to 0.  the order doesn't matter.  

For setYaw, it seemed cleaner to zero the yaw in the navx and then set our offset rather than trying to keep track of delta offsets from one set command to another.

Reply:
for manualInputInterfaces, I think it stays like it is.  zeroGyroscope is the externally-facing method.